uint32 object_id
---
srs_env_model_msgs/PlaneDesc plane
